frontend/apps/reader/modules/readerfooter.lua

```diff
@@ -377,8 +377,10 @@ local footerTextGeneratorMap = {
     page_turning_inverted = function(footer)
         local symbol_type = footer.settings.item_prefix
         local prefix = symbol_prefix[symbol_type].page_turning_inverted
-        if G_reader_settings:isTrue("input_invert_page_turn_keys") or G_reader_settings:isTrue("input_invert_left_page_turn_keys") or
-           G_reader_settings:isTrue("input_invert_right_page_turn_keys") or G_reader_settings:isTrue("inverse_reading_order") then
+        if G_reader_settings:isTrue("input_invert_page_turn_keys") or
+           G_reader_settings:isTrue("input_invert_left_page_turn_keys") or
+           G_reader_settings:isTrue("input_invert_right_page_turn_keys") or
+           footer.ui.view.inverse_reading_order then
```

Comment:
Can be `footer.view.inverse_reading_order`.
https://github.com/koreader/koreader/blob/698fc35c3f1a10233cf69085a87fb912e2f79841/frontend/apps/reader/modules/readerview.lua#L127-L130